Sources/eppo/EppoClient.swift

```diff
@@ -449,6 +445,7 @@ public class EppoClient {
         case defaultAllocationNull = "DEFAULT_ALLOCATION_NULL"
         case noActionsSuppliedForBandit = "NO_ACTIONS_SUPPLIED_FOR_BANDIT"
         case banditError = "BANDIT_ERROR"
+        case unknown = "UNKNOWN"
```

Comment:
https://github.com/Eppo-exp/eppo-ios-sdk/pull/61#discussion_r2102970735

> Keep it consistent by keeping the throws with masking behavior
Add a new return code of unknown for more correctness